README.md

```diff
@@ -76,3 +83,75 @@ attention to:
   Operator](https://github.com/luxas/kubeadm-workshop#deploying-the-prometheus-operator-for-monitoring-services-in-the-cluster)
 - [Setting up the custom metrics adapter and sample
   app](https://github.com/luxas/kubeadm-workshop#deploying-a-custom-metrics-api-server-and-a-sample-app)
+
+FAQs
+----
+
+### Why do my metrics keep jumping between a normal value and a very large number?
+
+You're probably switching between whole numbers (e.g. `10`) and milli-quantities (e.g. `10500m`).
+Worry not!  This is just how Kubernetes represents fractional values.  See the
+[Quantity Values](/docs/walkthrough.md#quantity-values) section of the walkthrough for a bit more
+information.
+
+### Why isn't my metric showing up?
+
+First, check your configuration.  Does it select your metric?  You can find the
+[default configuration](/deploy/custom-metrics-config-map.yaml) in the deploy directory,
+and more information about configuring the adapter in the [docs](/docs/config.md).
+
+Next, check if the discovery information looks right.  You should see the metrics showing
+up as associated with the resources you expect at. `/apis/custom.metrics.k8s.io/v1beta1/`.
+If not, make sure your series are labeled correctly.  Consumers of the custom metrics API
+(especially the HPA) don't do any special logic to associate a particular resource to
+a particular series, so you have to make sure that the adapter does it instead.
+
+For example, if you want a series `foo` to be associated with deployment `bar` in namespace
+`somens`, make sure there's some label that represents deployment name, and that the adapter
+is configured to use it.  With the default config, that means you'd need the query
+`foo{namespace="somens",deployment="bar"}` to return some results in Prometheus.
+
+Next, try using the `--v=6` flag on the adapter to see the exact queries being made
+by the adapter.  Try url-decoding the query and pasting it into the Prometheus web
+console to see if the query looks wrong.
+
+### My query contains multiple metrics, how do I make that work?
+
+It's actually fairly straightforward, if a bit non-obvious.  Simply choose one
+metric to act as the "discovery" and "naming" metric, and use that to configure
+the "discovery" and "naming" parts of the configuration.  Then, you can write
+whichever metrics you want in the `metricsQuery`!  The series query can contain
+whichever metrics you want, as long as they have the right set of labels.
+
+For example, if you have two metrics `foo_total` and `foo_count`, you might write
```

Comment:
It could be useful for clarity to say that `foo_total` has the label `system_name` which represents the actual `node`